lab-ron/app/component/app-gallery/app-gallery.html

```diff
@@ -0,0 +1,17 @@
+<section class="app-gallery">
+
+  <form novalidate ng-submit="appGalleryCtrl.addGalleryItem()">
+    <label for="title">title: </label>
```

Comment:
I the 'for' attributes value is supposed to be the id of the element it is a label for, not the name.